projects/lib/src/interceptors/default-oauth.interceptor.ts

```diff
@@ -39,7 +47,7 @@ export class DefaultOAuthInterceptor implements HttpInterceptor {
         if (!this.moduleConfig.resourceServer) {
             return next.handle(req);
         }
-        if (this.moduleConfig.resourceServer.allowedUrls && !this.checkUrl(url)) {
+        if (!this.checkUrl(url)) {
```

Comment:
I don't see this change after the merge of the PR, the final code still has
`if (this.moduleConfig.resourceServer.allowedUrls && !this.checkUrl(url)) {`
I need to set both `allowedUrls` and `customUrlValidation` to make it work

Reply:
PR created #584